rapids-cmake/cmake/support_conda_env.cmake

```diff
@@ -76,12 +76,45 @@ function(rapids_cmake_support_conda_env target)
 
   if(in_conda_build OR in_conda_prefix)
 
+    macro(modify_cmake_prefix_path_global)
+      cmake_parse_arguments("" "" "" "PATHS" ${ARGN})
+
+      if(NOT ("$ENV{CMAKE_PREFIX_PATH}" STREQUAL ""))
+        # If both CMAKE_PREFIX_PATH cmake and environment variables are populated, ensure the
+        # environment variable's paths are preserved in the cmake variable
+        cmake_path(CONVERT "$ENV{CMAKE_PREFIX_PATH}" TO_CMAKE_PATH_LIST _paths NORMALIZE)
+        list(PREPEND _PATHS ${_paths})
+      endif()
+
+      list(APPEND CMAKE_PREFIX_PATH ${_PATHS})
+      list(REMOVE_DUPLICATES CMAKE_PREFIX_PATH)
+      set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} PARENT_SCOPE)
+      message(VERBOSE "CMAKE_PREFIX_PATH set to: ${CMAKE_PREFIX_PATH}")
+    endmacro()
+
+    macro(modify_cmake_prefix_path_envvar)
+      cmake_parse_arguments("" "" "" "PATHS" ${ARGN})
+      cmake_path(CONVERT "$ENV{CMAKE_PREFIX_PATH}" TO_CMAKE_PATH_LIST _paths NORMALIZE)
+      list(APPEND _paths ${_PATHS})
```

Comment:
Can we make the variable usage between `modify_cmake_prefix_path_global` and `modify_cmake_prefix_path_envvar` consistent. It looks like in `modify_cmake_prefix_path_envvar` we append to `_paths` instead of `_PATHS`.

Reply:
The local `_paths` is the version of the `$ENV{CMAKE_PREFIX_PATH}` converted to a list and normalized. `_PATHS` is the argument list of paths to append.

Reply:
Ahhhhhh.